source/configure/plugins-configuration-settings.rst

```diff
@@ -915,6 +915,23 @@ Call recording quality
 
 .. |note| replace:: .
 
+Enable call ringing (Beta)
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. config:setting:: plugins-enablecallringing
+  :displayname: Enable call ringing (Plugins - Calls)
+  :systemconsole: Plugins > Calls
+  :configjson: N/A
+  :environment: N/A
+  :description: Enable or disable incoming call desktop alerts and ringing notifications
+
++--------------------------------------------------------------------------+--------------------------------------------------+
+| - **true**: Ringing functionality is enabled. Direct and group message   | - System Config path: **Plugins > Calls**        |
```

Comment:
@cpoile - Does this new Enable Ringing config setting have an equivalent ``config.json`` and environment variable value to include in the docs?

Reply:
Yep, added it above and below. No environment variable.